src/runtime_strings.js

```diff
@@ -187,16 +187,19 @@ function stringToUTF8(str, outPtr, maxBytesToWrite) {
 
 // Returns the number of bytes the given Javascript string takes if encoded as a UTF8 byte array, EXCLUDING the null terminator byte.
 function lengthBytesUTF8(str) {
-  var len = 0;
```

Comment:
How does this work without a len declared at all?

Reply:
Oh, good catch! It's works. I must have accidentally erased it